### PR TITLE
Show frame number in controls

### PR DESCRIPTION
**Pull request by subdavis**

### CODE REVIEW

File: client/dive-common/components/ControlsContainer.vue [hunk 1]
```diff
@@ -201,6 +201,21 @@ export default defineComponent({
...
+          <template #activator="{ on }">
+            <v-icon
+              small
+              class="mx-2"
+              v-on="on"
+            >
+              mdi-information
+            </v-icon>
+          </template>
+          <span>annotation framerate may be subsampled. frame numbers start at zero.</span>
```
Comment (BryonLewis) [inline client/dive-common/components/ControlsContainer.vue:217]: `<BR>` this into two lines?

File: server/dive_utils/constants.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 webValidImageFormats = {"png", "jpg", "jpeg"}
 validImageFormats = {*webValidImageFormats, "tif", "tiff", "sgi", "bmp", "pgm"}
-validVideoFormats = {"mp4", "avi", "mov", "mpg", "mpeg", "wmv", "flv"}
+validVideoFormats = {"mp4", "avi", "mov", "mpg", "mpeg", "wmv", "flv", "webm"}
```
Comment (BryonLewis) [inline server/dive_utils/constants.py:11]: I notice there are some differences with:
https://github.com/Kitware/dive/blob/main/client/dive-common/constants.ts
(probably my fault)
Should we see this as an opportunity to sync them up so we have the same mime and file extension support?

**Review by BryonLewis — CHANGES_REQUESTED** [review]
Just some minor questions/suggestions.

There is an alternative idea that I did on MEVA because I've seen it done elsewhere in other applications.  That was to make the time indicator/filename display a toggle where you click it and it switches to the  `currentFrame/totalFrames` instead of displaying them both at the same time.  I don't feel that strongly about it so just wanted to throw that out there as an option to reduce display if needed.  I don't know how important it is to have the time/filename and the frame number displayed at the exact same time or if the visual real estate is unnecessary for this feature..

File: server/dive_utils/constants.py (hunk 2 above)
Comment (subdavis) [reply server/dive_utils/constants.py:11]: Sure, I'll do that.

**Review by BryonLewis — APPROVED** [review]

**Pull request merged**